### DISCUSSION TITLE
Multiply a mask with a MRI

### DISCUSSION THREAD

**Discussion by gkrisp98** [TorchIO-project/torchio#827] [Q&A] [answered]
Hi, is there a method to multiply the mask and the MRI from a torchio Subject like those:
![sa](https://user-images.githubusercontent.com/57015579/155434388-d95cd893-e738-4fcf-a0b1-1ce110ef65a7.PNG)
t

**Comment by romainVala** [comment, marked as answer]
yes
https://torchio.readthedocs.io/transforms/preprocessing.html#mask
